src/test/java/seedu/address/model/person/NameTest.java

```diff
@@ -37,4 +36,18 @@ public void isValidName() {
         assertTrue(Name.isValidName("Capital Tan")); // with capital letters
         assertTrue(Name.isValidName("David Roger Jackson Ray Jr 2nd")); // long names
     }
+
+    @Test
+    public void capitalise() {
```

Comment:
I like that you account for many cases. Does adding additional space at the back of the input name also result in ```"Alex"```? Maybe we can have a test case for that to confirm the behaviour of the method?

Reply:
Good point